.eslintrc.js

```diff
@@ -53,6 +53,9 @@ module.exports = {
       {
         // These types are browser-provided, so trust that they exist.
         'definedTypes': [
+          'MediaCapabilities',
+          'MediaCapabilitiesDecodingInfo',
+          'MediaDecodingConfiguration',
```

Comment:
`MediaCapabilitiesDecodingInfo` and `MediaDecodingConfiguration` are dictionary types and not actually global variables.  IIUC eslint doesn't check the closure types, so this field is more about global variables and not actual types.

Reply:
> IIUC eslint doesn't check the closure types, so this field is more about global variables and not actual types.

I'm not sure if I understand your feedback.  Are you saying there's another way to get eslint to understand that they are defined externally?  Without these lines, it complains that I haven't defined the types.